src/Backend/WASMWrapper.js

```diff
@@ -198,7 +198,7 @@ export default class WASMWrapper {
             [...ids, _energies.byteOffset, _stoppingPowers.byteOffset]
         )
 
-        const energies = !err ? Array.from(_energies).map(energy => energy.toFixed(6)) : [0]
+        const energies = !err ? Array.from(_energies).map(energy => energy.toFixed(6).replace(/(\d)0+$/gm, '$1')) : [0]
```

Comment:
Regular expressions are not self-explanatory. The purpose of using them here is not clear to code reader.
Add an appropriate comment explaining:
 - what is the reason behind using such code
 - what this code is doing